nvflare/client/tracking.py

```diff
@@ -17,10 +17,19 @@
 from nvflare.apis.analytix import AnalyticsDataType
 from nvflare.app_common.tracking.tracker_types import LogWriterName
 
+# flake8: noqa
+from .api import default_context as default_context
 from .api import log
+from .api_context import APIContext
 
 
-class SummaryWriter:
+class _BaseWriter:
+    def __init__(self, ctx: Optional[APIContext] = None):
+        global default_context
```

Comment:
Should minimize use of gloabl.
You can do: ctx = api.get_default_ctx().

Reply:
Thanks for the detail review, change as suggested